src/commands/cork_proposal/aave_v2_stablecoin/initiate_shutdown.rs

```diff
@@ -37,7 +40,7 @@ pub struct InitiateShutdownCmd {
 impl Runnable for InitiateShutdownCmd {
     fn run(&self) {
         abscissa_tokio::run_with_actix(&APP, async {
-            cellars::validate_cellar_id(&self.cellar_id)
+            cellars::validate_cellar_id(self.chain_id, &self.cellar_id)
```

Comment:
Just commenting here since it applies to all of the commands:

`print_proposal` works for `ScheduledCorkProposal`, but `AxelarScheduledCorkProposal` has a chain ID field